webfrontend/src/components/popup/ChangeNameForm.scss

```diff
@@ -0,0 +1,59 @@
+.form-container {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100vh;
```

Comment:
height should not be full height, since there is the nav bar taking up space as well. Ideally, if the page height is naturally not enough to be scrollable, then do not set the height such that the page is scrollable.